requirements-jax-cuda.txt

```diff
@@ -10,6 +10,6 @@ torchvision>=0.16.0
 # Jax with cuda support.
 # TODO: 0.4.24 has an updated Cuda version breaks Jax CI.
```

Comment:
Can we update this comment to say that "Keep same version as in Keras Repo?"

Reply:
done!